sdk/core/core-websockets/package.json

```diff
@@ -0,0 +1,132 @@
+  "sideEffects": false,
+  "prettier": "@azure/eslint-plugin-azure-sdk/prettier.json",
+  "scripts": {
+    "build": "npm run clean && dev-tool run build-package && dev-tool run extract-api",
+    "build:samples": "echo Obsolete",
+    "build:test": "echo skipped. actual commands inlined in browser test scripts",
+    "check-format": "dev-tool run vendored prettier --list-different --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.{ts,cts,mts}\" \"test/**/*.{ts,cts,mts}\" \"*.{js,cjs,mjs,json}\"",
+    "clean": "dev-tool run vendored rimraf --glob dist dist-* temp types *.tgz *.log",
+    "execute:samples": "dev-tool samples run samples-dev",
+    "extract-api": "dev-tool run build-package && dev-tool run extract-api",
+    "format": "dev-tool run vendored prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.{ts,cts,mts}\" \"test/**/*.{ts,cts,mts}\" \"samples-dev/**/*.{ts,cts,mts}\" \"*.{js,cjs,mjs,json}\"",
+    "integration-test": "npm run integration-test:node && npm run integration-test:browser",
+    "integration-test:browser": "echo skipped",
+    "integration-test:node": "echo skipped",
+    "lint": "eslint README.md package.json api-extractor.json src test",
+    "lint:fix": "eslint README.md package.json api-extractor.json src test --fix --fix-type [problem,suggestion]",
+    "pack": "npm pack 2>&1",
+    "start-server": "tsx test/server/start.mts",
+    "test": "npm run clean && dev-tool run build-package && npm run unit-test:node && dev-tool run build-test && npm run unit-test:browser && npm run integration-test",
+    "test:browser": "npm run clean && npm run build && npm run integration-test:browser",
+    "test:node": "npm run clean && dev-tool run build-package && npm run integration-test:node",
+    "unit-test": "npm run unit-test:node && npm run unit-test:browser",
+    "unit-test:browser": "npm run clean && dev-tool run build-package && dev-tool run build-test && dev-tool run test:vitest --no-test-proxy --browser",
+    "unit-test:node": "dev-tool run test:vitest --no-test-proxy",
+    "update-snippets": "dev-tool run update-snippets"
+  },
+  "devDependencies": {
+    "@azure-tools/test-utils-vitest": "^1.0.0",
+    "@azure-tools/vite-plugin-browser-test-map": "^1.0.0",
+    "@azure/dev-tool": "^1.0.0",
+    "@azure/eslint-plugin-azure-sdk": "^3.0.0",
+    "@types/chai-as-promised": "^8.0.1",
+    "@types/node": "^18.0.0",
+    "@types/node-forge": "^1.3.11",
+    "@vitest/browser": "^3.0.4",
+    "@vitest/coverage-istanbul": "^3.0.4",
+    "chai-as-promised": "^8.0.1",
+    "dotenv": "^16.4.7",
+    "eslint": "^9.19.0",
+    "node-forge": "^1.3.1",
+    "playwright": "^1.50.0",
+    "typescript": "~5.7.3",
+    "vitest": "^3.0.4"
+  },
+  "dependencies": {
+    "@azure/core-util": "^1.11.0",
+    "@azure/logger": "^1.1.4",
+    "@types/ws": "^8.5.14",
```

Comment:
Should the types here be dependencies or just dev dependencies?  As stated in the review.md, I'm not a fan of exposing their underlying types without some sort of abstraction layer which makes it easier for us to switch implementations and not be beholden to one library's implementation.